docs_user/modules/proc_migrating-ovn-data.adoc

```diff
@@ -212,25 +210,27 @@ spec:
 '
 ----
 
-. If you'd like to run OVN gateway services on OCP nodes, also enable the control plane `ovn-controller` service:
+. Optional: To run OVN gateway services on {OpenShiftShort} nodes, enable the control plane `ovn-controller` service:
```

Comment:
Step 11: Is this step actually optional? Or is it conditional, i.e., it only applies to customers who have OVN gateway services on OCP nodes?

Reply:
the latter, it only applies to the users who have OVN gateways on OCP nodes